app/code/Magento/InventoryCatalog/Plugin/Inventory/Reservation/UpdateLegacyCatalogInventoryPlugin.php

```diff
@@ -56,20 +58,20 @@ public function __construct(
      * @param void $result
      * @param ReservationInterface[] $reservations
      *
+     * @return void
      * @see ReservationsAppendInterface::execute
      * @SuppressWarnings(PHPMD.UnusedFormalParameter)
-     * @return void
      */
     public function afterExecute(ReservationsAppendInterface $subject, $result, array $reservations)
     {
         $this->updateStockItemAndStatusTable($reservations);
-        return $result;
     }
 
     /**
      * Updates cataloginventory_stock_item and cataloginventory_stock_status qty with reservation information.
      *
      * @param ReservationInterface[] $reservations
+     *
      * @return void
      */
     private function updateStockItemAndStatusTable(array $reservations)
```

Comment:
Looks like we can move this code in `afterExecute`
We don't have advantage in separate private method in this case